app/controllers/contacts_controller.rb

```diff
@@ -0,0 +1,51 @@
+class ContactsController < ApplicationController
+  def create
```

Comment:
this is a lot of instance variables - can we cut down on the number of these we're setting?